fs/grub/shared.h

```diff
@@ -371,6 +371,9 @@ extern char *grub_scratch_mem;
 #define strcpy grub_strcpy
 #endif /* WITHOUT_LIBC_STUBS */
 
+#define SECTOR_SIZE 512
```

Comment:
This already exists in https://github.com/XboxDev/cromwell/blob/4b2db9caf5ef0e75a1a98a62b945ba9d8a3665e3/fs/grub/shared.h#L172

Reply:
Indeed, fixed.